Session/Conversations/ConversationVC.swift

```diff
@@ -465,30 +466,53 @@ final class ConversationVC : BaseVC, ConversationViewModelDelegate, OWSConversat
     // MARK: Updating
     
     func updateNavBarButtons() {
+        // Back button (to appear on pushed screen)
+        let backButton = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        backButton.tintColor = Colors.text
+        navigationItem.backBarButtonItem = backButton
```

Comment:
Can we do this in `BaseVC` so we can remove all back button text of the VCs inheriting `BaseVC`?